### PR TITLE
backend/accounts: test ExportCSV

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: backend/accounts/baseaccount_test.go [hunk 1]
```diff
@@ -103,4 +109,44 @@ func TestBaseAccount(t *testing.T) {
...
+		const header = "Time,Type,Amount,Unit,Fee,Address,Transaction ID,Note\n"
+
+		require.Equal(t, header, export(nil))
+
+		require.NoError(t, account.notes.SetTxNote("some-internal-tx-id", "some note, with a comma"))
+		fee := coin.NewAmountFromInt64(101)
+		require.Equal(t,
+			header+
+				",sent,123,satoshi,101,some-address,some-tx-id,\"some note, with a comma\"\n"+
+				",sent_to_yourself,456,satoshi,,another-address,some-tx-id,\"some note, with a comma\"\n",
```
Comment (x1ddos) [inline backend/accounts/baseaccount_test.go:129]: You can also use the backticks for slightly nicer reading:

```
`,sent_to_self,456,satoshi,,another-address,some-tx-id,"some note, with a comma"\n`
```

although not sure about `\n`. might not be interpreted correctly but worth a try.

Altertnatively,

```go
const body = `,sent,123,...
,sent_to_self,456,...
`
```

(no need for `\n`)

File: backend/accounts/baseaccount_test.go [hunk 2]
```diff
@@ -103,4 +109,44 @@ func TestBaseAccount(t *testing.T) {
...
+
+		const header = "Time,Type,Amount,Unit,Fee,Address,Transaction ID,Note\n"
+
+		require.Equal(t, header, export(nil))
+
+		require.NoError(t, account.notes.SetTxNote("some-internal-tx-id", "some note, with a comma"))
+		fee := coin.NewAmountFromInt64(101)
+		require.Equal(t,
+			header+
+				",sent,123,satoshi,101,some-address,some-tx-id,\"some note, with a comma\"\n"+
```
Comment (x1ddos) [inline backend/accounts/baseaccount_test.go:128]: would've been nice to also test the timestamp.

File: backend/accounts/baseaccount_test.go (hunk 1 above)
Comment (benma) [reply backend/accounts/baseaccount_test.go]: :+1: done

File: backend/accounts/baseaccount_test.go (hunk 2 above)
Comment (benma) [reply backend/accounts/baseaccount_test.go]: Done

**Pull request merged**